std/json.zig

```diff
@@ -0,0 +1,1308 @@
+// JSON parser conforming to RFC8259.
+//
+// https://tools.ietf.org/html/rfc8259
+
+const std = @import("index.zig");
+const mem = std.mem;
+
+const u1 = @IntType(false, 1);
+const u256 = @IntType(false, 256);
+
+pub const TokenId = enum(u5) {
```

Comment:
is the `u5` meant to save space? zig will use the minimum tag size if you do not specify. This feature is more meant for `packed` structs where you want to control the bit layout.